src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.csproj

```diff
@@ -17,6 +18,7 @@
     <AndroidGeneratedClassDirectory Condition=" '$(AndroidGeneratedClassDirectory)' == '' ">..\..\src\Mono.Android\obj\$(Configuration)\monoandroid10\android-$(AndroidLatestStablePlatformId)\mcw</AndroidGeneratedClassDirectory>
     <LangVersion>latest</LangVersion>
     <NoWarn>8632</NoWarn>
+    <GenerateResxSource>true</GenerateResxSource>
```

Comment:
It's not directly related to this source line, but one nice feature of `GenerateResxSource` is that it can optionally create methods like `string FormatXA0001 (object p0)` for resources that contain format strings. So after this initial setup PR for `GenerateResxSource` is merged, Xamarin.Android could consider enabling that by setting `$(GenerateResxSourceEmitFormatMethods)`=`true` and updating uses of resources containing format items to use the new generated methods.